e2e/cypress/integration/api/iam/policies_api.spec.ts

```diff
@@ -29,78 +54,210 @@ describe('policies API', () => {
   });
 
   after(() => {
-    cy.cleanupV2IAMObjectsByIDPrefixes(cypressPrefix, ['policies', 'projects']);
+    cy.cleanupIAMObjectsByIDPrefixes(cypressPrefix, ['policies', 'projects']);
   });
 
-  describe('POST /apis/iam/v2/policies', () => {
-    beforeEach(() => {
-      cy.cleanupV2IAMObjectsByIDPrefixes(cypressPrefix, ['policies']);
+  describe('policies API', () => {
+
+    before(() => {
+      cy.cleanupIAMObjectsByIDPrefixes(cypressPrefix, ['policies']);
     });
 
-    afterEach(() => {
-      cy.cleanupV2IAMObjectsByIDPrefixes(cypressPrefix, ['policies']);
+    after(() => {
+      cy.cleanupIAMObjectsByIDPrefixes(cypressPrefix, ['policies']);
     });
 
-    it('returns 400 when there are no statements',  () => {
-      cy.request({
-        ...defaultAdminReq,
-        method: 'POST',
-        failOnStatusCode: false,
-        body: {
-          id: `${cypressPrefix}-policy-${now}`,
-          name: `${cypressPrefix} policy ${now}`,
-          members: ['team:local:test'],
-          projects: [project1.id]
-        }
-      }).then((response) => {
-       expect(response.status).to.equal(400);
+    describe('POST /apis/iam/v2/policies', () => {
+
+      it('returns 200 when all valid inputs are provided', () => {
+        cy.request({
+          ...defaultAdminReq,
+          method: 'POST',
+          failOnStatusCode: false,
+          body: {
+            id: policyID,
+            name: policyName,
+            members: ['team:local:test'],
+            statements: [
+              {
+                effect: 'ALLOW',
+                actions: ['test:svc:someaction', 'test:svc:otheraction'],
+                projects: [project1.id]
+              }
+            ],
+            projects: [project1.id]
+          }
+        }).then((response) => {
+          expect(response.status).to.equal(200);
```

Comment:
Fine if you want to postpone this to another PR, but would be nice to start using named constants:
```suggestion
          expect(response.status).to.equal(HttpStatus.OK);
```
All we need to do is add this to e2e/cypress/support/constants.ts:
```
export enum HttpStatus {
  OK = 200,
  BAD_REQUEST = 400,
  FORBIDDEN = 403,
  NOT_FOUND = 404,
  CONFLICT = 409,
  PRECONDITION_FAILED = 412,
  INTERNAL_SERVER_ERROR = 500
}
```

Reply:
Yeah I think this is worth addressing in a different issue, since it will touch all of the cypress files.